src/app/mock-pets.ts

```diff
@@ -4,29 +4,129 @@ export const PETS: Pet[] = [
     {
         id: 1,
         name: 'Cassidy',
-        birthday: '11-12-2019',
```

Comment:
If you create a petsService class then it can use this mock PETS internally and hide it from everything else in your project.  Everything else in your project will be referencing your petsService.  Eventually you will update your petsService to use an API and if it is done correctly, the rest of your application will not need to be changed.